tutorials/frontend/build_gcn.py

```diff
@@ -0,0 +1,282 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Build a Graph Convolutional Network
+=====================
+**Author**: `Yulun Yao <https://yulunyao.io/>`_
+
+This article is an introductory tutorial to build a Graph Convolutional Network (GCN) with Relay.
```

Comment:
tutorial to seems a bit strange. tutorial on/tutorial for?

Reply:
tutorial on building

Thanks for pointing out typo